src/GoogleCloudStorageServiceProvider.php

```diff
@@ -78,6 +76,28 @@ public function boot()
         });
     }
 
+    /**
+     * Create a new StorageClient
+     *
+     * @param  mixed $config
+     * @return \Google\Cloud\Storage\StorageClient
+     */
+    private function createClient($config)
+    {
+        if (is_string($config['key_file'])) {
```

Comment:
This will break if `key_file` key is not present in config.
```suggestion
        if (is_string(array_get($config, 'key_file'))) {
```